angular/projects/admin-nrpti/src/app/import/import.component.ts

```diff
@@ -143,18 +142,19 @@ export class ImportComponent implements OnInit {
 
   checkChange() {}
 
-  startJob() {
+  async startJob() {
+    await this.startTask();
+    this.showAlert = true;
     // tslint:disable-next-line: no-this-assignment
     const self = this;
```

Comment:
Is the ```self=this``` necessary here?

Since arrow functions don't have their own scope wouldn't ```this.showAlert``` use the 'this' of the current import class?